tests/test_Optimiser.py

```diff
@@ -74,14 +74,38 @@ def test_optimiser_with_fixed_nodes(self):
               initial_membership=[2, 1, 0])
       # Equivalent to setting initial membership
       #partition.set_membership([2, 1, 2])
-      opt = leidenalg.Optimiser()
       fixed_nodes = [True, False, False]
-      opt.optimise_partition(partition, fixed_nodes=fixed_nodes)
+      original_quality = partition.quality()
+      diff = self.optimiser.optimise_partition(partition, fixed_nodes=fixed_nodes)
```

Comment:
Good catch that it should be using `self.optimiser` instead of defining a new `Optimiser` object.